src/bootstrap_gui.cpp

```diff
@@ -102,7 +102,8 @@ class BootstrapErrorWindow : public Window {
 	{
 		if (widget == WID_BEM_MESSAGE) {
 			*size = GetStringBoundingBox(STR_MISSING_GRAPHICS_ERROR);
-			size->height = GetStringHeight(STR_MISSING_GRAPHICS_ERROR, size->width - WidgetDimensions::scaled.frametext.Horizontal()) + WidgetDimensions::scaled.frametext.Vertical();
+			size->width += WidgetDimensions::scaled.frametext.Horizontal();
+			size->height = GetStringHeight(STR_MISSING_GRAPHICS_ERROR, size->width) + WidgetDimensions::scaled.frametext.Vertical();
```

Comment:
At this point, size already has the string height too, so...
```suggestion
size->height += WidgetDimensions::scaled.frametext.Vertical();
```